assets/airports/essa.json

```diff
@@ -0,0 +1,3852 @@
+            "rwy": {
+                "ESSA19R": []
+            },
+            "draw": [
+                ["XILAN", "SA482", "SA481", "SA480", "SA476", "OXINU"]
+            ]
+        },
+        "XILAN3V": {
+            "icao": "XILAN3V",
+            "name": "Xilan Three Victor",
+            "entryPoints": {
+                "XILAN": ["XILAN"]
+            },
+            "body": [["SA494", "A80+"], ["SA493", "A50+"], "SA492", ["EKDAS", "A25+"]],
+            "rwy": {
+                "ESSA26": []
+            },
+            "draw": [
+                ["XILAN", "SA494", "SA493", "SA492", "EKDAS"]
+            ]
+        },
+        "XILAN4M": {
+            "icao": "XILAN4M",
+            "name": "Xilan Four Mike",
+            "holds": {
+                "XILAN": "069|left|1.5min|S230-",
+                "TEB"  : "178|right|1min|S230-"
+            },
+            "entryPoints": {
+                "XILAN": [["XILAN", "A150-"]]
+            },
+            "body": ["SA570", "@TEB"],
+            "rwy": {
+                "ESSA01L": [],
+                "ESSA01R": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB"]
+            ]
+        },
+        "XILAN4S": {
+            "icao": "XILAN4S",
+            "name": "Xilan Four Sierra",
+            "holds": {
+                "XILAN": "069|left|1.5min|S230-",
+                "TEB"  : "178|right|1min|S230-"
+            },
+            "entryPoints": {
+                "XILAN": [["XILAN", "A150-"]]
+            },
+            "body": ["SA570", "@TEB"],
+            "rwy": {
+                "ESSA08": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB"]
+            ]
+        },
+        "XILAN5P": {
+            "icao": "XILAN5P",
+            "name": "Xilan Five Papa",
+            "holds": {
+                "XILAN": "069|left|1.5min|S230-",
+                "ERK"  : "038|left|1min|S230-"
+            },
+            "entryPoints": {
+                "XILAN": [["XILAN", "A150-"]]
+            },
+            "body": ["SA570", "TEB", "@ERK"],
+            "rwy": {
+                "ESSA19R": [],
+                "ESSA19L": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB", "ERK"]
+            ]
+        },
+        "XILAN5T": {
+            "icao": "XILAN5T",
+            "name": "Xilan Five Tango",
+            "holds": {
+                "XILAN": "069|left|1.5min|S230-",
+                "TEB"  : "178|right|1min|S230-"
+            },
+            "entryPoints": {
+                "XILAN": [["XILAN", "A120-"]]
+            },
+            "body": ["SA570", "@TEB"],
+            "rwy": {
+                "ESSA26": []
+            },
+            "draw": [
+                ["XILAN", "SA570", "TEB"]
+            ]
+        }
+    },
+    "_comment": "A spreadsheet with the calculations used for spawnPatterns is attached to issue #1695 on Github",
```

Comment:
This spreadsheet is found at the bottom of the description of issue #1695 if any reviewer is interested. :)